src/main/default/lwc/provideDetailsScreen/provideDetailsScreen.html

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="slds-var-p-vertical_xx-large container">
+    <div class="content">
+      <div class="slds-is-relative screen-header">
+        <c-flow-progress-indicator
+          current-step="provide details"
+        ></c-flow-progress-indicator>
+        <div class="slds-is-absolute navigation">
+          <button onclick={handleBack} class="slds-button">
+            <lightning-icon
+              class="slds-button__icon_left previous-button-icon"
+              size="x-small"
+              icon-name="utility:chevronleft"
+              alternative-text={labels.previous}
+            ></lightning-icon>
+            {labels.previous}
+          </button>
+        </div>
+      </div>
+      <div class="slds-var-p-top_medium slds-var-m-top_xx-large">
+        <c-main-title
+          align-center
+          title={labels.provideDetailsTitle}
+          subtitle={labels.provideDetailsSubtitle}
+        ></c-main-title>
+      </div>
+      <div class="slds-var-p-top_medium">
+        <div class="slds-var-m-top_xx-large">
+          <c-quantic-case-classification
+            required
+            initial-value={priority}
+            title="priority"
+            label={labels.priority}
+            coveo-field-name="sfpriority"
+            max-suggestions="2"
+            sf-field-api-name="Priority"
+            engine-id={engineId}
+            select-placeholder={labels.moreOptions}
+          >
+          </c-quantic-case-classification>
+        </div>
+        <div class="slds-var-m-top_xx-large">
+          <c-quantic-case-classification
+            required
+            initial-value={origin}
+            title="origin"
+            label={labels.origin}
+            coveo-field-name="sforigin"
+            max-suggestions="2"
+            sf-field-api-name="Origin"
+            engine-id={engineId}
+            select-placeholder={labels.moreOptions}
+          ></c-quantic-case-classification>
+        </div>
+        <!-- <div class="slds-var-m-top_xx-large">
+          <c-quantic-case-classification
+            initial-value={reason}
+            title="reason"
+            label={labels.reason}
+            coveo-field-name="sfreason"
+            max-suggestions="2"
+            sf-field-api-name="Reason"
+            engine-id={engineId}
+            select-placeholder={labels.moreOptions}
+          ></c-quantic-case-classification>
+        </div> -->
```

Comment:
Why is this field commented out?

Reply:
Perhaps we could add some explanation at the beginning of the comment. It currently seems to be dead code.

Reply:
Yes definitely!